pkg/dashboard/templates/dashboard.gohtml

```diff
@@ -156,5 +159,11 @@
       {{ template "footer" . }}
     </footer>
   </div>
+  <script>
+    window.INSIGHTS_HOST = "{{ .Data.InsightsHost }}"
+  </script>
+  <script defer src="static/js/email.js"></script>
+  <script defer src="static/js/api-token.js"></script>
+  <script defer src="static/js/cost_settings.js"></script>
```

Comment:
Putting these here is an outdated pattern that may affect performance negatively on "large sites" which may be triggered when we have a large amount of data.

> An old-fashioned solution to this problem used to be to put your script element right at the bottom of the body (e.g. just before the </body> tag), so that it would load after all the HTML has been parsed. The problem with this solution is that loading/parsing of the script is completely blocked until the HTML DOM has been loaded. On larger sites with lots of JavaScript, this can cause a major performance issue, slowing down your site. 

From: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/What_is_JavaScript#script_loading_strategies